m4_sessions/1810-inning/boundaries.md

```diff
@@ -1,31 +1,33 @@
-## Group Members:
+## Group Members: LeeLee Graham, Seth Schwartz, Keegan Corrigan, Cody Taft
 
-
-
-## Name of Session: 
+## Name of Session: Setting Healthy Boundaries
 
 ## Session Description (Why should students come to this session): (EX: Are you interested in biological enhancement? Drop in to learn about biohacking.)
 
-## Session Outcomes 
+-A discussion oriented workshop with an emphasis on developing tools to set appropriate and healthy boundaries at work.
 
-(What will students learn/leave with?): 
+## Session Outcomes
 
+(What will students learn/leave with?): 
 
+- The "no" has to go somewhere.
+- Using "no" as a currency. Exercising "no".
+- They leave with bags and bags of "no's"
+- You become the things you say 'yes' to. 
 
      Best practices:  No section should be longer than 20 minutes
      Create a session that YOU would want to attend
      Share resources in your slack channel
      All sessions NEED to have a closing activity, reflection, discussion
+     
 ## Session Outline 
 
-(Please include who is facilitating and how much time you are going to spend in each section):
+- Brief Introduction
 
-## Recommended Trajectory 
+- Small group discussion about last time we set boundaries and how it made us feel, talking candidly about what makes us uncomfortable.  (We will break out into 2 groups. 2 facilitators per group.)
```

Comment:
This sounds good...maybe starting with some norms for the day:
1) Take chances
2) Push yourself
3) Be honest

or something like that